hw/xbox/xid.c

```diff
@@ -112,7 +112,7 @@ static const USBDescIface desc_iface_xbox_gamepad = {
     .bInterfaceProtocol            = 0x00,
     .eps = (USBDescEndpoint[]) {
         {
-            .bEndpointAddress      = USB_DIR_IN | 0x02,
+            .bEndpointAddress      = USB_DIR_IN | 0x01,
```

Comment:
this comes from my Controller S. Need to review using Duke

Reply:
My Duke reports ```bEndpointAddress     0x02  EP 2 OUT ```

Reply:
cool, yeah - I'll also look at a duke still. I plan to run linux in QEMU and then compare using libusb [with a real gamepad + virtual gamepad connected at once, sending both the same requests]